documents/2-RTB/Documentazione_Interna/Norme_di_Progetto.typ

```diff
@@ -118,3 +129,182 @@ Il team si aspetta di rispettare in ogni momento gli standard di qualità defini
+- Creare Test per verifica e validazione del codice
+- Scrivere il #glossary("manuale utente")
+
+==== Ticketing
+
+Il Team utilizza il sistema di #glossary("Issue Tracking Sistem") (ITS) di GitHub. Questo permette di assegnare compiti e ruoli ai membri del Team, di monitorare l'avanzamento del progetto e di tenere traccia delle attività svolte, oltre ad essere facilmente collegabile a #glossary("Pull Request") e #glossary("Commit").
+
+Le issue principali sono create dal responsabile del gruppo che assegna i compiti ai vari membri del team. In seguito, ogni problema è ulteriormente suddiviso in sotto-problemi e di conseguenza, "sotto-issue". Queste ultime sono create non dal Responsabile, ma dall'assegnatario della issue principale. Questo permette di avere una visione chiara e dettagliata delle attività da svolgere e di monitorare l'avanzamento del progetto. Ad esempio:
+- Responsabile crea la Issue `Norme di Progetto` e la assegna al componente X
+- X studia il documento `Norme di progetto` e individua sotto-issue per rendere il problemapiù approcciabile.
+- X crea le sotto-issue individutae, le auto-asssegna e sceglie il verificatore.
+
+Ogni Issue è composta da:
+- Titolo: titolo della issue
+- Descrizione: descrizione dell'attività da completare
+- Assegnatario (o Assegnatari): chi deve svolgere questa issue
+- Verificatore: chi deve verificare il lavoro svoltop
+- #glossary("Milestone"): traguardo da raggiungere
+- #glossary("Label"): etichetta per categorizzare le issue
+- #glossary("Project"): progetto a cui appartiene la issue
+- stato: aperta, chiusa, in corso, ecc.
+- data di inizio: data di inizio dell'attività
+- data di fine: data di fine dell'attività
+
+La creazione e la gestione della issue segue il seguente flusso:
+
+1. Creazione della Issue
+2. Assegnazione della Issue
+3. Creazione delle sotto-issue
+4. Assegnazione delle sotto-issue
+5. Inizio dell'attività
+6. Spostamento della Issue da `To Do` a `In Progress`
+7. Svolgimento attività
+8. Fine attività
+9. Creazione Pull Request
+10. Verifica del lavoro svolto
+  1. se corretto, si passa al punto 11
+  2. se non corretto, il verificatore lascia commenti sulla PR riguardanti le modifiche necessarie, l'assegnatario della issue apporta le modifiche e si ripete il punto 9
+11. Chiusura della Issue
+12. Spostamento della Issue da `In Progress` a `Done`
+13. Chiusura PR e merge su branch `Develop`
+14. Eventuale eliminazione del branch (solo se non deve essere riutilizzato)
+
+*Milestones* \
+Il gruppo, dopo una pianificazione accurata, ha deciso di impostare le milestones ad "artefatti" e non a "sprint".
+Il gruppo ritiene che le Milestones indichino qualcosa di completato, quindi è stato deciso di ritenere una milestone completata solamente quando un artefatto è stato completato e verificato. Al contrario, è stata impostata una data di inizio e una data di fine alla singola issue, di modo che sia comunque monitorabile nel tempo l'avanzamento del progetto.
```

Comment:
268:
di modo che sia comunque monitorabile nel tempo l'avanzamento del progetto. -> in modo che l'avanzamento del progetto sia comunque monitorabile nel tempo.

Reply:
done